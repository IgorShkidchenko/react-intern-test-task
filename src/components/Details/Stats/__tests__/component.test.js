import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import StatsComponent from "../component";

configure({ adapter: new Adapter() });

describe("<StatsComponent />", () => {
  it("renders correctly", () => {
    const repository = { watchers: 1, stargazers_count: 1, forks: 1 };
    const wrapper = shallow(<StatsComponent repository={repository} />);

    expect(wrapper).toMatchSnapshot();
  });
});
