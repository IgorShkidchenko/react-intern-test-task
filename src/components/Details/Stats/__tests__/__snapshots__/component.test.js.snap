// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StatsComponent /> renders correctly 1`] = `
<table>
  <thead>
    <tr>
      <td>
        Stats:
      </td>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>
        watchers: 
        1
      </td>
    </tr>
    <tr>
      <td>
        stars: 
        1
      </td>
    </tr>
    <tr>
      <td>
        forks: 
        1
      </td>
    </tr>
  </tbody>
</table>
`;
